app/controllers/posts_controller.rb

```diff
@@ -27,6 +27,13 @@ def show
     @like = Like.new
   end
 
+  def destroy
+    @post = Post.find(params[:id])
+    authorize! :destroy, @post
+    @post.destroy
+    redirect_to user_posts_path(current_user), notice: 'Post was successfully deleted.'
+  end
+
```

Comment:
- [ ] In line 2 of this page, it means before defining all methods, please 🙏🏽 the `load_and_authorize_resource` method, which is provided to automatically authorize all actions in a RESTful style resource controller. It will use a before action to load the resource into an instance variable and authorize it for every action. You can check more about this on the official [guide](https://github.com/CanCanCommunity/cancancan).

Reply:
Hi @franclobo,
Thank you for letting me know the  " load_and_authorize_resource " helper. it reduces time and workload.